<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Aggregator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/icons.qrc">
    <normaloff>:/icons/a_icon.png</normaloff>:/icons/a_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frameBtn">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnAggregate">
         <property name="text">
          <string>Aggregate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddFormulaNode">
         <property name="text">
          <string>Add Formula Node</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnAddDataNode">
         <property name="text">
          <string>Add Data Node</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemoveNode">
         <property name="text">
          <string>Remove Node</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Select Tree</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetInp" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QFrame" name="frame_1">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="maximumSize">
             <size>
              <width>80000</width>
              <height>900</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="autoScrollMargin">
             <number>22</number>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="autoExpandDelay">
             <number>-1</number>
            </property>
            <property name="indentation">
             <number>22</number>
            </property>
            <property name="rootIsDecorated">
             <bool>true</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>false</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>false</bool>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <attribute name="headerCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>70</number>
            </attribute>
            <attribute name="headerHighlightSections">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Variable</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Formula</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Source</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Result</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <item>
              <widget class="QLabel" name="icon">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item alignment="Qt::AlignTop">
              <widget class="QTableWidget" name="table">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>220</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>220</height>
                </size>
               </property>
               <property name="autoFillBackground">
                <bool>true</bool>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
               </property>
               <property name="autoScroll">
                <bool>true</bool>
               </property>
               <property name="alternatingRowColors">
                <bool>false</bool>
               </property>
               <property name="showGrid">
                <bool>true</bool>
               </property>
               <property name="gridStyle">
                <enum>Qt::DashDotDotLine</enum>
               </property>
               <property name="cornerButtonEnabled">
                <bool>true</bool>
               </property>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>40</number>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>80</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>23</number>
               </attribute>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <row>
                <property name="text">
                 <string/>
                </property>
               </row>
               <column>
                <property name="text">
                 <string>Property</string>
                </property>
                <property name="textAlignment">
                 <set>AlignJustify|AlignVCenter</set>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Value</string>
                </property>
               </column>
               <item row="0" column="0">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="0" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="1" column="0">
                <property name="text">
                 <string>Name</string>
                </property>
                <property name="toolTip">
                 <string/>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <underline>false</underline>
                 </font>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="1" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="2" column="0">
                <property name="text">
                 <string>Variable</string>
                </property>
                <property name="toolTip">
                 <string>Set a custom variable</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="2" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="3" column="0">
                <property name="text">
                 <string>Formula</string>
                </property>
                <property name="toolTip">
                 <string>Set aggregation formula</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="3" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="4" column="0">
                <property name="text">
                 <string>Source</string>
                </property>
                <property name="toolTip">
                 <string>Database table and column ex: table.col</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="4" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="5" column="0">
                <property name="text">
                 <string>Criteria</string>
                </property>
                <property name="toolTip">
                 <string>Normalize accordingly</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="5" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="6" column="0">
                <property name="text">
                 <string>Normalize</string>
                </property>
                <property name="toolTip">
                 <string>Automatic or Custom</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="6" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="7" column="0">
                <property name="text">
                 <string>Min</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled|ItemIsTristate</set>
                </property>
               </item>
               <item row="7" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="8" column="0">
                <property name="text">
                 <string>Max</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                  <strikeout>false</strikeout>
                 </font>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled|ItemIsTristate</set>
                </property>
               </item>
               <item row="8" column="1">
                <property name="text">
                 <string/>
                </property>
               </item>
               <item row="9" column="0">
                <property name="text">
                 <string>Weights</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="flags">
                 <set>ItemIsEnabled</set>
                </property>
               </item>
               <item row="9" column="1">
                <property name="text">
                 <string/>
                </property>
                <property name="textAlignment">
                 <set>AlignCenter</set>
                </property>
               </item>
               <item row="10" column="0">
                <property name="text">
                 <string>Alpha</string>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <italic>true</italic>
                  <bold>true</bold>
                 </font>
                </property>
               </item>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
              <widget class="QTableWidget" name="weightTable">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>160</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>250</height>
                </size>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="gridStyle">
                <enum>Qt::DashDotDotLine</enum>
               </property>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>60</number>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>70</number>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Name</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Manuel</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Auto</string>
                </property>
               </column>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="status"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDatabase"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Add Connection</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
